src/Employer/Employer.Web/Orchestrators/Part1/EmployerOrchestrator.cs

```diff
@@ -105,5 +106,16 @@ private LocationOrganisationViewModel MapLegalEntitiesToOrgs(LegalEntity data)
         {
             return new LocationOrganisationViewModel { Id = data.LegalEntityId.ToString(), Name = data.Name };
         }
+        
+        private IEnumerable<LocationOrganisationViewModel> BuildLegalEntityViewModels(EditVacancyInfo info, string employerAccountId)
+        {
+            if (info == null || !info.LegalEntities.Any())
+            {
+                Logger.LogError("No legal entities found for {employerAccountId}", employerAccountId);
+                return null; // TODO: Can we carry on without a list of legal entities.
```

Comment:
How about we do something on the start of the session where we check the user is associated to an employer account with at least one legal entity. If there are none then we redirect them to specific page (not 403) instructing them to add legal entities. Not strictly associated with this bit of code but something to prevent the case. Chris has made some changes to do with starting sessions on #134 .

Reply:
There will always be at least one Organisation when querying the account api. This scenario is more if the querystore hasn't been updated with that information.